cves/CVE-2017-14056.yml

```diff
@@ -64,7 +64,17 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  When parsing the header of a RL2 video file, the system receives
+  a value for the amount of frames the video will have. It then goes through
+  each frame to get additional data such as chunk size to help with the decoding
+  process. However, there is no check for an end of file character in these loops,
+  which could lead to a potential Denial of Service attack. An attacker could 
+  modify the number of frames in the header to a very large number, which 
+  would cause the program to loop through this process much more than intended.
+  Since the program needs to save the chunk and audio frame data, a bunch of 
```

Comment:
This is well written! It's informative and explains the issue thoroughly without using too much technical jargon. When you refer to parsing, is this in encoding? Specifying that would help me a bit.

One small nit: `would` -> `could` for most of this to keep the tenses and phrasing similar throughout.